src/metpy/calc/thermo.py

```diff
@@ -45,7 +45,8 @@ def relative_humidity_from_dewpoint(temperature, dewpoint):
     -------
     `pint.Quantity`
         Relative humidity
-
+        
```

Comment:
Can you add a Notes heading here like:
```suggestion
        Notes
        _____

```

Also be sure to make sure any blank lines don't have any trailing spaces at the end (which is what Codacy was complaining about.).